build.sbt

```diff
@@ -10,17 +10,18 @@ crossScalaVersions := Seq("2.10.5", "2.11.7")
 
 spName := "databricks/spark-avro"
 
-sparkVersion := "1.3.0"
-
-sparkComponents += "sql"
+sparkVersion := "1.4.0"
```

Comment:
Can you add a matrix to the README that lists spark version -> library version?

1.3 -> 1.0.0
1.4+ -> 1.1.0-SNAPSHOT